src/codeu/chat/common/ConversationHeader.java

```diff
@@ -63,4 +63,21 @@ public ConversationHeader(Uuid id, Uuid owner, Time creation, String title) {
     this.title = title;
 
   }
+
+  @Override
+  public boolean equals(Object toCompare)
+  {
+    if(!(toCompare instanceof ConversationHeader))
+      return false;
+    ConversationHeader toCompareConv = (ConversationHeader)(toCompare);
+    if(!this.id.equals(toCompareConv.id))
```

Comment:
Is it possible for `this.id` to be `null`? Similar question for all of the `if` statements in this PR.